docs/cristin.ttl

```diff
@@ -16,35 +16,55 @@
 :Published
     a               rdfs:Class ;
     rdfs:label      "PUBLISHED" ;
+    rdfs:label      "PUBLISERT" ;
     rdfs:subClassOf :PublicationStatus .
 
+:InPress
+    a               rdfs:Class ;
+    rdfs:label      "IN_PRESS" ;
+    rdfs:subClassOf :PublicationStatus .
+
+::TestPublicationStatus
+    a               rdfs:Class ;
+    rdfs:label      "TEST" ;
+    rdfs:subClassOf :PublicationStatus .
+
+
 :VARBEID_SPRAK_Class
     a            rdfs:Class ;
     rdfs:comment "The structure defining the VARBEID_SPRAK property." .
 
 :VARBEID_PERSON_Class
     a            rdfs:Class ;
     rdfs:comment """
-    The structure of the VARARBEID_PERSON entry. It contains information about the status of the contributor
-    at the time the Result was registered in Cristin.
+    The structure of the VARARBEID_PERSON entry. It contains user submitted information about the status of the contributor
+    at the time the Result was registered in Cristin. Some of the information is provided by the user manually in order to reflect the
+    user's name at the time the manuscript/publication was submitted to the publisher (e.g. Elsevier).
     """ .
 
 :VARBEID_PERSON_STED_Class
     a            rdfs:Class ;
     rdfs:comment """
     The structure of the VARBEID_PERSON_STED entry.
     It contains information about the status of a contributor's institution at the time the Result
-    was registered in Cristin. It is uncertain whether the contributor presently belongs to the same institution,
-    or if the institution status has changed.
+    was registered in Cristin. If the institution which the contributor belonged when the Cristin entry was created has now changed,
+    then this will be reflected in the entry. However the fact that the  contributor has moved to another institution is not reflected here.
+
+    Example:
+    Contributor C worked for for institution "InstA" when they registered publication P in Cristin.
+    "InstA" has merged with another institution and now is called "InstB".
+    Contributor C is working today for "InstC". The Cristin entry will indicate the fact that:
+    "When Contributor C was affiliated with InstB, they published P".
```

Comment:
How about "Contributor C published P representing InstB", but even so the information that they originally represented InstA is preserved alonside (in the "..._opprinnelig" fields).

Maybe it's best not to put these examples up here in the class. How about that the class "... contains information about the contributor's institutional affiliation represented in the result entity at the time of publication, as well as the current instance of that institution, should it have evolved over time."